### PR TITLE
Added netstandard1.6 to frameworks of installation dummy project

### PR DESCRIPTION
**Pull request by neilhuiz**
Fix for issue #575

### CODE REVIEW

**Comment by gregg-miskelly** [comment]
@neilhuiz Thanks! Looks good to me. @chuckries do you see any issues with this?

**Comment by chuckries** [comment]
Works for me. I'm curious why this new import is needed now. Is it due to a newer vesion of CLI tools? I noticed in the issue that @neilhuiz is on a preview 3 version of dotnet CLI.

**Comment by neilhuiz** [comment]
@chuckries the issue repros with 1.0.0-rc2-002665 as well.

```
.NET Command Line Tools (1.0.0-rc2-002665)

Product Information:
 Version:     1.0.0-rc2-002665
 Commit Sha:  6eea734681

Runtime Environment:
 OS Name:     Windows
 OS Version:  6.3.9600
 OS Platform: Windows
 RID:         win81-x64
```

**Comment by gregg-miskelly** [comment]
LGTM

File: src/coreclr-debug/install.ts [hunk 1]
```diff
@@ -183,7 +183,8 @@ export class DebugInstaller
                 "System.ComponentModel.Annotations":  "4.1.0",  
                 "System.ComponentModel.EventBasedAsync":  "4.0.11",
                 "System.Runtime.Serialization.Primitives": "4.1.1",
-                "System.Net.Http":  "4.1.0"
+                "System.Net.Http":  "4.1.0",
+	            "System.Reflection.Emit.Lightweight": "4.0.0"
```
Comment (gregg-miskelly) [inline src/coreclr-debug/install.ts]: nuget.org lists the current version of this as 4.0.1. Does this still work if you use that instead?

**Comment by DustinCampbell** [comment]
@chuckries, @gregg-miskelly: Is this ready to merge?

**Comment by chuckries** [comment]
@gregg-miskelly did we decide that this was necessary or not?

**Comment by gregg-miskelly** [comment]
@neilhuiz was going to try and dig some more to understand why his machine doesn't think that netcore1.0 isn't using netstandard1.6. Neil, did you have any luck?

**Comment by neilhuiz** [comment]
Just got done checking on this. It turns out that a preview 1 build of the CLI tools was the root cause. Once I uninstalled it, the debugger setup and execution. I apologizes for the kerfuffle on this; turns out it was environmental, not a problem with the product.

**Comment by DustinCampbell** [comment]
No worries -- thanks for digging in to it!

**Pull request closed without merging**